src/WinRT.Runtime/ComWrappersSupport.cs

```diff
@@ -343,12 +343,13 @@ private static bool ShouldProvideIReference(Type type)
         {
             static bool IsWindowsRuntimeType(Type type)
```

Comment:
Does WinRT.Projections.IsTypeWindowsRuntimeType do the equivalent of this function?